pkgs/python-packages/joinmarket/default.nix

```
@@ -0,0 +1,104 @@
+{ version
+, src
+, lib
+, buildPythonPackageWithDepsCheck
+, pythonOlder
+, pythonAtLeast
+, pythonRelaxDepsHook
+, pytestCheckHook
+, setuptools
+, fetchurl
+, chromalog
+, cryptography
+, service-identity
+, twisted
+, txtorcon
+, python-bitcointx
+, argon2_cffi
+, autobahn
+, bencoderpyx
+, klein
+, mnemonic
+, pyjwt
+, werkzeug
+, libnacl
+, pyopenssl
+}:
+
+buildPythonPackageWithDepsCheck rec {
+  pname = "joinmarket";
+  inherit version src;
+  format = "pyproject";
+
+  # Since v0.9.11, Python older than v3.8 is not supported. Python v3.12 is
+  # still not supported.
+  disabled = (pythonOlder "3.8") || (pythonAtLeast "3.13");
```

Judging by your comment, this should be correct:
```suggestion
  disabled = (pythonOlder "3.8") || (pythonAtLeast "3.12");
```